components/NavLink.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Link from 'next/link'
+
+type NavLinkProps = {
+  text: string
+  path: string
+  activePath?: string
+  as?: string
+  internal?: true
+  blank?: true
+  className?: string
+}
+
+const NavLink: React.FC<NavLinkProps> = ({
+  text,
+  path,
+  activePath,
+  as,
+  internal,
+  blank,
+  className
+}) => {
+  const active = path === activePath
+  className = className ? className : ''
```

Comment:
what happens if you passed in className directly?

Reply:
I was getting className="undefined" without this line

Reply:
then `className = className || ''` is clearer than a tertiary

Reply:
fixed!